enterprise/migrations/0233_create_vw_enterprise_customer_ecus_pecus.py

```diff
@@ -0,0 +1,56 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('enterprise', '0232_backfill_ecu_table_user_fk_data_migration'),
+    ]
+    operations = [
+        # Custom SQL to add view that unions EnterpriseCustomerUser and PendingEnterpriseCustomerUser records
+        # together to facilitate querying both simultaneously
+        migrations.RunSQL(
+            sql="""
+                CREATE VIEW view_enterprise_customer_support_users AS
+                    SELECT ecu.enterprise_customer_id,
+                    au.email AS user_email,
+                    FALSE AS is_pending,
+                    ecu.user_id,
+                    au.username,
+                    au.first_name,
```

Comment:
I don't think we should be using the first_name and last_name from auth_user, but instead the name field from auth_userprofile with first and last name as a backup like [here](https://github.com/openedx/edx-enterprise/blob/e5c210fe559f094bc37e10ec3bf9df663033c6a2/enterprise/models.py#L4741)

Reply:
Good call out!  For this work I'm aiming for data parity with the old model since it's used by support tools, and we don't use `first_name` and `last_name` for our logic, so I've added this as[ follow-up work.](https://2u-internal.atlassian.net/browse/ENT-10350)